uk.co.compucorp.civicrm.hrcomments/README.md

```diff
@@ -0,0 +1,16 @@
+HR Comments extension
+======
+
+This extension provides Commenting functionality for CiviHR .
+Any CiviHR extension can use the comment functionality by invoking
+the API's provided by this extension.
+
+Installation
+------
+
+Go to 
+- Administer -> System Settings -> Manage Extensions
+
+and install CiviHR Core (uk.co.compucorp.civicrm.hrcomments) extension.
```

Comment:
I think you based yourself on the HR Core extension, right? The text here is still mentioning CiviHR Core.